index.html

```diff
@@ -10,16 +10,17 @@
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <h1 id="logo">Ari karim</h1>
+      <button id="menuu" type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
+        <span id="span1" class="span1">close</span>
```

Comment:
**Required**

- [ ] The `close` button/tap should only be visible on mobile views. At the moment, the button is visible on desktop views. Since this can only work when the user accesses the app on mobile view, I suggest you hide the button on large screen sizes.
- [ ] Also, from the project requirement, this should be a `button`, and not a `span` tag. This is important for accessibility purposes.

Reply:
This is issue was successfully addressed :heavy_check_mark:

Reply:
- [ ] [Optional] This is totally optional, you can style the close button and make it look like the given mockup,